pkg/render/manager.go

```diff
@@ -970,6 +975,52 @@ func (c *managerComponent) managerAllowTigeraNetworkPolicy() *v3.NetworkPolicy {
 	}
 }
 
+func (c *managerComponent) multiTenantManagedClustersAccess() []client.Object {
+	var objects []client.Object
+	objects = append(objects, &rbacv1.Role{
```

Comment:
In multi-tenant systems, there is already going to be a role in this namespace - I don't think we should have multiple roles per-tenant that allow managed cluster acccess.

Reply:
I am only adding a RoleBinding in the tenant namespace.